src/sentry/static/sentry/app/utils/discover/discoverQuery.tsx

```diff
@@ -18,6 +18,7 @@ type Props = {
   location: Location;
   eventView: EventView;
   orgSlug: string;
+  extraSort?: string;
```

Comment:
The `EventView` object looks like it supports multiple sorts. Are we not able to use them here?

Reply:
When i tried to use that, I noticed that the UI would show that both fields were sorted (eg. RPM+transactions on load), Which i thought would be more confusing, since the subsort can't be changed.

Also the existing code only grabs the first sort when there are multiple:
[`encodeSort(this.sorts[0])`](https://github.com/getsentry/sentry/blob/9e8672832c976e946dece57890f342c8bc36542b/src/sentry/static/sentry/app/utils/discover/eventView.tsx#L886)
[`const sort = sorts.find(currentSort => sortKeys.includes(currentSort.field));`](https://github.com/getsentry/sentry/blob/9e8672832c976e946dece57890f342c8bc36542b/src/sentry/static/sentry/app/utils/discover/eventView.tsx#L283)

Reply:
`EventView` doesn't really support multiple sorts. 

The passing of the sort array was due to type plumbing since the `sort` query string can exist multiple times (for example from the URL: `?sort=this&sort=that`).

We'd have to change the semantics to make an assumption of subsorting if we're given an array of sorts that has at least two items.